clusterman/autoscaler/autoscaler.py

```diff
@@ -181,7 +181,7 @@ def run(self, dry_run: bool = False, timestamp: Optional[arrow.Arrow] = None) ->
             self.target_capacity_gauge.set(new_target_capacity, {"dry_run": dry_run})
             self.max_capacity_gauge.set(
                 self.pool_manager.max_capacity,
-                {"dry_run": dry_run, "alert_on_max_capacity": self.pool_manager.alert_on_max_capacity},
+                {"dry_run": dry_run, "alert_on_max_capacity": self.pool_manager.alert_on_max_capacity, "team": self.pool_manager.pool_owner},
```

Comment:
should we `s/team/pool_owner` to keep things consistent between config and metric labeling?

Reply:
oh, that's fine then - might be good to add that in a comment here tho since i'm not sure how many people are aware of that (or maybe i'm just the only one that didn't know this :p)